tests/source/expr.rs

```diff
@@ -244,3 +244,15 @@ fn issue767() {
     } else if let false = false {
     }
 }
+
+fn ranges() {
+    let x = aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa .. bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb;
+    let y = aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa ... bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb;
+    let z = ... x ;
+    let infi_range_2 = ... ;
+
+    a ... b
+
+    // the expr below won't compile for some reason...
+    // let a = 0 ... ;
```

Comment:
I commented on #893, but this isn't meant to compile, so the comment can be removed.